frontend/src/components/Calendar/KFCalendar.jsx

```diff
@@ -68,7 +75,12 @@ const KFCalendar = () => {
       week: { header: DayViewHeader, event: WeekEvent },
       month: { event: MonthEvent },
       toolbar: (props) => (
-        <KFToolbar {...props} activeView={activeView} setActiveView={setActiveView} />
+        <KFToolbar
+          {...props}
+          activeView={activeView}
+          setActiveView={setActiveView}
```

Comment:
Actually it would be better to refactor this piece of code as in React we actually shouldn't pass setState as a prop. [explanation](https://medium.com/@christopherthai/why-you-shouldnt-pass-react-s-setstate-as-a-prop-a-deep-dive-8a3dcd74bec8#:~:text=Passing%20the%20setState%20down%20as,is%20getting%20updated%20or%20changed. ) Actually I don't who the author was, if this person was not you or anyone else available who worked on it, we can move it to a separate issue, @S-chahine

Reply:
I see, this happens in many places in the app...

Reply:
This is already a huge PR - we can make a separate ticket for it. As refactoring might take much time, maybe, better do it separately in this case.